src/serac/numerics/functional/boundary_integral.hpp

```diff
@@ -48,25 +48,28 @@ class BoundaryIntegral {
    * @param[in] X The actual (not reference) coordinates of all quadrature points
    * @param[in] N The unit normals of all quadrature points
    * @param[in] qf The user-provided quadrature function
-   * @param[in] arg_indices indices used to select which trail spaces to use in evaluation kernels
+   * @param[in] active_arguments indices used to select which trail spaces to use in evaluation kernels
```

Comment:
typo: `trail` -> `trial`

Reply:
oops, I got the one in the domain_integral, but missed this one